src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.js

```diff
@@ -2,31 +2,61 @@ import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import { Link } from 'react-router-dom';
 
-export const BannerUpgrade = ({ currentPlan, currentPlanList }) => {
-  const getBannerInfo = (type) => {
+export const BannerUpgrade = ({ sessionPlan, currentPlan, currentPlanList }) => {
+  const getUpgradeInfo = (type) => {
     const bannerInfo = { messageId: `plan_calculator.banner_for_${type.replace('-', '_')}` };
+    const suggestionInfo = {
+      messageId: `plan_calculator.suggestion_for_${type.replace('-', '_')}`,
+    };
+
     switch (type) {
       case 'prepaid':
       case 'subscribers':
         return {
-          ...bannerInfo,
-          link: `/plan-selection/${currentPlan.featureSet}/monthly-deliveries`,
+          banner: {
+            ...bannerInfo,
+            link: `/plan-selection/${currentPlan.featureSet}/monthly-deliveries`,
+          },
+          suggestion: {
+            ...suggestionInfo,
+            link: '#',
+          },
         };
       case 'monthly-deliveries':
         // TODO: define where to go in this case
-        return { ...bannerInfo, link: `/email-marketing-agencies` };
+        return {
+          banner: {
+            ...bannerInfo,
+            link: `/email-marketing-agencies`,
+          },
+          suggestion: {
+            ...suggestionInfo,
+            link: '#',
+          },
+        };
       default:
-        return { ...bannerInfo, messageId: `plan_calculator.banner_for_unknown` };
+        return {
+          banner: {
+            ...bannerInfo,
+            messageId: `plan_calculator.banner_for_unknown`,
+          },
+          suggestion: {
+            ...suggestionInfo,
+            messageId: `plan_calculator.suggestion_for_unknown`,
+          },
+        };
     }
   };
 
-  if (currentPlan.id === currentPlanList[currentPlanList.length - 1].id) {
-    const bannerInfo = getBannerInfo(currentPlan.type);
+  const hightestPlan = currentPlanList.length === 1 && currentPlan.id === sessionPlan.idPlan;
+
+  if (currentPlan.id === currentPlanList[currentPlanList.length - 1].id && !hightestPlan) {
```

Comment:
Perhaps this could be set into a variabel `currentPlan.id === currentPlanList[currentPlanList.length - 1].id`
```suggestion
  const currentPlanSelected = currentPlan.id === currentPlanList[currentPlanList.length - 1].id;
  if (currentPlanSelected && !hightestPlan) {
```